README.md

```diff
@@ -1 +1,25 @@
 # Badges Initiative
+
+This is the organizational repository for the Node.js Badges Strategic Initiative, as chartered under the [Community Committee].
+
+This initiative is tasked with facilitating the design, development and production of Node.js Open Badges and associated content.
+
+These responsibilities include, but are not limited to:
+ - Management of the Node.js [Open Badges] account;
+ - The new badge approval and release process;
+ - Asset creation and style guide standards;
+ - Badge award processes;
+ - Development and Maintenance of badge issuance automation tooling;
+ - Production and distribution of stickers, pins, and related products created from content that emerges from this initiative.
+
+## Governance
+This initiative adopts the general Node.js [Code of Conduct], as well as its own
+[Contributing Guidelines].
+
+## Team
+- [@amiller-gh](https://github.com/amiller-gh) - **Adam Miller**, Community Committee Champion
```

Comment:
Its lonely here! Consider this PR an open call for new friends who want to actively work on this project 👍 

Tagging all participants from the inaugural issue here: @Bamieh, @bnb, @oe, @Tiriel, @rachelnicole, @gr2m, @mhdawson

Reply:
Count me in!

Reply:
Sorry, I left this too long aside.
@amiller-gh Do you want me to submit a second PR to add myself when this one is merged, or do you prefer to add me now? If the latter, the name's Benjamin Zaslavsky!

Reply:
Nope! I got ya. Sorry for the delay, busy week 👍